VirtoCommerce.LiquidThemeEngine/SassFileManager.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.IO;
+using LibSassHost;
+using VirtoCommerce.Storefront.Model;
+using VirtoCommerce.Storefront.Model.Common;
+using VirtoCommerce.Storefront.Model.StaticContent;
+
+namespace VirtoCommerce.LiquidThemeEngine
+{
+    public class SassFileManager: ISassFileManager
+    {
+        private readonly IContentBlobProvider _contentBlobProvider;
+
+        public bool SupportsConversionToAbsolutePath { get; } = false;
+
+        public string CurrentDirectory { get; set; }
+
+        public SassFileManager(IContentBlobProvider contentBlobProvider)
+        {
+            _contentBlobProvider = contentBlobProvider;
+        }
+
+        public string GetCurrentDirectory() => CurrentDirectory;
+
+        public bool FileExists(string path)
+        {
+            // Workaround for directories
+            if (string.IsNullOrEmpty(Path.GetExtension(path)))
+            {
+                return false;
+            }
+            return _contentBlobProvider.PathExists(path);
```

Comment:
cosmetic note: we have a stylecop rule which required add empty space after closed brackets. 
see: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1513.md
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1508.md

we going to use stylecop in all backend projects so it can be simply corrected now